commonmark/src/main/java/org/commonmark/renderer/html/HtmlNodeRendererContext.java

```diff
@@ -44,4 +46,16 @@
      * @return whether HTML blocks and tags should be escaped or not
      */
     boolean shouldEscapeHtml();
+
+    /**
+     *
+     * @return true if the {@link UrlSanitizer} should be used.
+     */
+    boolean shouldBeSafe();
```

Comment:
Let's use a more descriptive name for this, how about `shouldSanitizeUrls`?

Reply:
Are you suggesting the property should be renamed (eg: `builder.sanitizeUrls(true)` instead of `builder.save(true)`), or just this function?

Reply:
All the related methods, yeah.